_posts/2020-02-07-welcome-mc-boucher.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Marie-Christine Boucher joins the Programming Historian Project Team"
+authors:
+- Sofia Papastamkou
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/avatars/marie-christine-boucher.png" alt="Portrait photograph of Marie-Christine Boucher"/><figcaption>Marie-Christine Boucher has joined the Programming Historian</figcaption></figure></p>
+
+We have the pleasure to announce that Marie-Christine Boucher has joined the editorial board of *The Programming Historian*!
+
+Marie-Cristine is a doctoral candidate in Comparative Literature and Cultural Studies at Justus-Liebig University, Giessen, Germany. She holds B.A. and M.A. degrees in German Studies from the Université de Montréal, Québec, Canada. She has a background in Library and Information Studies, where she first discovered her interest in web development, programming, and use of digital technologies in humanities. She has been acquiring skills by self-learning and attending Digital Humanities training courses. Currently, she is in charge of web and IT administration at the International Graduate Centre for the Study of Culture of the Justus-Liebig University. Furthermore, she is also a member of the editorial board of the open access journal [On_Culture](https://www.on-culture.org/). 
```

Comment:
Line 13: Christine (the 'h' is missing)

Reply:
I would tend to say 'use of digital technologies in **the** humanities' - can a native English speaker help us there?

Reply:
She holds **her** B.A.

Reply:
... use of digital technologies in humanities, **and has since been acquiring**  ... 

(I feel like otherwise it can sound like it's any skills)